core/src/main/java/com/netflix/conductor/core/config/Configuration.java

```diff
@@ -64,16 +64,44 @@
 
     String ADDITIONAL_MODULES_PROPERTY_NAME = "conductor.additional.modules";
 
+    String EXECUTION_LOCK_ENABLED_PROPERTY_NAME = "decider.locking.enabled";
+    boolean EXECUTION_LOCK_ENABLED_DEFAULT_VALUE = false;
+
+    String LOCKING_SERVER_PROPERTY_NAME = "locking.server";
+    String LOCKING_SERVER_DEFAULT_VALUE = "noop_lock";
+
+    String IGNORE_LOCKING_EXCEPTIONS_PROPERTY_NAME = "locking.exceptions.ignore";
+    boolean IGNORE_LOCKING_EXCEPTIONS_DEFAULT_VALUE = false;
+
     //TODO add constants for input/output external payload related properties.
```

Comment:
do you need to add this?

Reply:
This helps to optionally ignore the locking exceptions (not the failures to obtain lock though) and not impact workflow executions.